pipeline/clinvar/clinvar.py

```diff
@@ -148,13 +219,13 @@ def __init__(self, element, debug=False):
 
         measureSet = element.find("MeasureSet")
         #if measureSet.get("Type") == "Variant":
-        if debug:
-            if len(measureSet.findall("Measure")) > 1:
-                print(self.id, "has multiple measures")
-        if len(measureSet.findall("Measure")) == 1:
+
+        if len(measureSet.findall("Measure")) > 1:
+            logging.warn("Assertion with ID " + str(self.id) + " has multiple measures. Taking first one.")
+        if len(measureSet.findall("Measure")) >= 1:
             name = measureSet.find("Name")
             if name == None:
-                variantName = none
+                variantName = None
```

Comment:
very possible that this is not used with the current datasets we are getting from clinvar. I've just changed it, as it was highlighted in my IDE, it would crash otherwise